js/api.js

```diff
@@ -0,0 +1,39 @@
+import {showAlert} from './util.js';
+
+const BASE_DATA_URL = 'https://24.javascript.pages.academy/keksobooking/data';
+const BASE_API_URL = 'https://24.javascript.pages.academy/keksobooking';
+
+const getData = (onSuccess) => {
+  fetch(BASE_DATA_URL)
+    .then((response) => response.json())
+    .then((offers) => {
+      onSuccess(offers);
+    })
+    .catch(() => {
+      showAlert('errorData');
+    });
+};
```

Comment:
```suggestion
const getData = (onSuccess) => {
  fetch(BASE_DATA_URL)
    .then((response) => response.json())
    .then((offers) => onSuccess(offers))
    .catch(() => showAlert('errorData'));
};
```
Функционально ничего не меняется, но как по мне - выглядит симпотичнее